core/src/test/java/org/elasticsearch/script/expression/ExpressionScriptTests.java

```diff
@@ -385,8 +388,63 @@ public void testStringSpecialValueVariable() throws Exception {
             message = e.toString();
         }
         assertThat(message + "should have contained ExpressionScriptExecutionException",
-                   message.contains("ExpressionScriptExecutionException"), equalTo(true));
+                message.contains("ExpressionScriptExecutionException"), equalTo(true));
         assertThat(message + "should have contained text variable error",
-                   message.contains("text variable"), equalTo(true));
+                message.contains("text variable"), equalTo(true));
+    }
+
+    public void testExecutableScripts() throws Exception {
```

Comment:
@jdconrad sure. The place where the expression script will be used is here[1]. Is this enough for what you need? If not, please let me know.

[1] https://github.com/elastic/elasticsearch/blob/a216062d88205c0d109ac00a432b45bc90a14fe8/core/src/main/java/org/elasticsearch/search/aggregations/pipeline/seriesarithmetic/SeriesArithmeticPipelineAggregator.java#L109-123